files/en-us/web/api/ext_texture_norm16/index.html

```diff
@@ -0,0 +1,108 @@
+---
+title: EXT_texture_norm16
+slug: Web/API/EXT_texture_norm16
+tags:
+  - API
+  - Reference
+  - WebGL
+  - WebGL extension
+---
+<div>{{APIRef("WebGL")}}</div>
+
+<p>The <code><strong>EXT_texture_norm16</strong></code> extension is part of the <a href="/en-US/docs/Web/API/WebGL_API">WebGL API</a> and provides a set of new 16-bit signed normalized and unsigned normalized fixed point texture, renderbuffer and texture buffer formats.</p>
```

Comment:
```suggestion
<p>The <code><strong>EXT_texture_norm16</strong></code> extension is part of the <a href="/en-US/docs/Web/API/WebGL_API">WebGL API</a> and provides a set of new 16-bit signed normalized and unsigned normalized formats (fixed-point texture, renderbuffer and texture buffer).</p>
```
I realize this text is from https://www.khronos.org/registry/webgl/extensions/EXT_texture_norm16/ but its a pretty big pile of adjectives upon adjectives, so maybe for MDN we can un-pile it a bit…